pensjon-brevbaker/src/test/kotlin/no/nav/pensjon/brev/Fixtures.kt

```diff
@@ -36,4 +40,226 @@ object Fixtures {
     )
 
     val fellesAuto = felles.copy(signerendeSaksbehandlere = null)
+
+    @Suppress("UNCHECKED_CAST")
+    fun <T: Any> create(letterDataType: KClass<T>): T =
```

Comment:
Dette kan bli veldig stort over tid. Kan vi skille de ut på noe vis?

Reply:
Eneste jeg kan se for meg er å lage en funksjon per klasse.